app/javascript/components/lease_ups/LeaseUpApplicationsPage.js

```diff
@@ -94,6 +98,13 @@ const LeaseUpApplicationsPage = () => {
     eagerPagination: new EagerPagination(ROWS_PER_PAGE, SERVER_PAGE_SIZE)
   })
 
+  useEffect(() => {
+    if (flagsReady && !partnersPaginationEnabled) {
+      state.eagerPagination = new EagerPagination(ROWS_PER_PAGE, 50000)
```

Comment:
issue: use `setState` instead of assignment when changing React states.

Reply:
Thank you!

Reply:
Following up on a pairing session we had:
We should use `setState({ eagerPagination: new EagerPagination(ROWS_PER_PAGE, 50000) })` here.
However, the `useAsync` function further down needs to add `state.eagerPagination` to it's dependency array so it will fetch things properly whenever the pagination values changes due to the flag.